frontend-shared/src/related-terms/index.ts

```diff
@@ -0,0 +1,174 @@
+import { Word2VecAPIContext, Word2VecAPIProvider } from "./provider";
+
+import h from "@macrostrat/hyper";
+import { useAPIResult, useAPIHelpers } from "@macrostrat/ui-components";
+import { useState } from "react";
+import { CollapseCard } from "~/shared/ui";
+import {
+  Button,
+  AnchorButton,
+  Intent,
+  Menu,
+  MenuDivider,
+  MenuItem,
+  Popover,
+} from "@blueprintjs/core";
+import { format } from "d3-format";
+import { findBestModel, joinWords } from "./util";
+
+const fmt = format(".3f");
+
+type WordResult = [string, number];
+
+const TermResult = (props: { data: WordResult }) => {
+  const [word, corr] = props.data;
+  return h("li", [
+    h("span.word", word.replace("_", " ")),
+    h("span.correlation", fmt(corr)),
+  ]);
+};
+
+const TermResults = (props: { words: WordResult[] | null }) => {
+  const { words } = props;
+  if (words == null || words.length == 0)
+    return h("p.no-results", "No results");
+  return h(
+    "ul.term-results",
+    words.map((d, i) => h(TermResult, { key: i, data: d }))
+  );
+};
+
+type WordRelatedTermsProps = {
+  word: string;
+  route?: string;
+  params?: object;
+  model: string;
+};
+
+const WordRelatedTerms = (props: WordRelatedTermsProps) => {
+  const {
+    word,
+    route = "most_similar",
+    params: baseParams = {},
+    model = "trigram_lowered_cleaned",
+  } = props;
+  const params = {
+    word: word.replace(" ", "_"),
+    model,
+    ...baseParams,
+  };
+
+  const res =
+    useAPIResult("most_similar", params, { context: Word2VecAPIContext }) ?? [];
+  const { buildURL } = useAPIHelpers();
+  const url = buildURL("most_similar", {
```

Comment:
Looks like this isn't building the URL correctly. It ends up linking out to:

https://xdddev.chtc.io/sets/xdd-covid-19/cosmos/apimost_similar?api_key=<REDECATED>&word=test&model=trigram_cleaned&n=50

When it should link to 

https://xdddev.chtc.io/sets/xdd-covid-19/word2vec/api/most_similar?word=test&model=trigram_cleaned&n=50


The actual call to the word2vec API works fine (and correctly doesn't appear to pass along the API key). Is the `useAPIHelpers()` call using a COSMOS context?

Reply:
Nice catch. Fix is exactly as you suggest.